src/Symfony/Component/DependencyInjection/IntrospectableContainerInterface.php

```diff
@@ -1,25 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\DependencyInjection;
-
-/**
- * IntrospectableContainerInterface defines additional introspection functionality
- * for containers, allowing logic to be implemented based on a Container's state.
- *
- * @author Evan Villemez <evillemez@gmail.com>
- *
- * @deprecated Since version 3.0, to be removed in 4.0. Use ContainerInterface
```

Comment:
It says deprecated since version 3? If that is true, it should not be removed. If it's not true, it needs to be fixed in earlier branches.